src/core/variable.jl

```diff
@@ -190,18 +190,18 @@ end
 
 "Create tap variables."
 function variable_mc_oltc_tap(pm::_PMs.AbstractPowerModel; nw::Int=pm.cnw, bounded=true)
-    nphases = 3
-    oltc_ids = _PMs.ids(pm, pm.cnw, :transformer)
-    for c in 1:nphases
-        _PMs.var(pm, nw, c)[:tap] = JuMP.@variable(pm.model,
-            [i in oltc_ids],
-            base_name="$(nw)_tm",
-            start=_PMs.ref(pm, nw, :transformer, i, "tm")[c]
+    nph = 3
+    for p in 1:nph
```

Comment:
should loop like `for p in _PMs.conductor_ids(pm)` for easier extensibility. If you want to fix to exactly three phases I would use something like `@assert length(_PMs.conductor_ids(pm) == 3`

Reply:
This was mostly done with the 4-wire scheme in mind, where phases would be different from conductors. But that would require further editing in any case, so I will implement it as you suggest everywhere for now.